### ISSUE TITLE
Loss is too low!

### ISSUE THREAD

**Issue by opza** [davidsandberg/facenet#665]
Hello
I was having problems implementing FaceNet using CNTK.
![image](https://user-images.githubusercontent.com/23049766/37497467-c13ab662-28fb-11e8-8691-fb45b8251601.png)

Using Softmax cross entropy loss, CASIA-Web was learned in advance and triplet loss was applied.
However, the loss is so low that accuracy is no longer increasing.
Why??
